### PR TITLE
0.9.0.Alpha2 release announcement

### PR DESCRIPTION
**Pull request by gunnarmorling**

### CODE REVIEW

File: blog/2018-10-04-debezium-0-9-0-alpha2-released.adoc [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+* support for table/collection names with special characters for MySQL (https://issues.jboss.org/browse/DBZ-878[DBZ-878]) and MongoDB (https://issues.jboss.org/browse/DBZ-865[DBZ-865]) and
+* fixed handling of blacklisted tables with the new Antlr-based DDL parser (https://issues.jboss.org/browse/DBZ-872[DBZ-872]).
+
+Community member Ian Axelrod provided a fix for a potential performance issue,
+where changes to tables with TOAST columns in Postgres would cause repeated updates to the connector's internal schema metadata,
+which can be a costly operation (https://issues.jboss.org/browse/DBZ-911[DBZ-911]).
+Please refer to the link:/docs/connectors/postgres/[Postgres connector documentation] for details on the new `schema.refresh.mode` option,
+which deals with this issue.
+
+In terms of version upgrades with migrated to the latest releases of the MySQL (https://issues.jboss.org/browse/DBZ-763[DBZ-763], https://issues.jboss.org/browse/DBZ-764[DBZ-764]) and Postgres drivers (https://issues.jboss.org/browse/DBZ-912[DBZ-912]).
```
Comment (jpechane) [inline blog/2018-10-04-debezium-0-9-0-alpha2-released.adoc]: Is this sentence correct?

Comment (gunnarmorling) [reply blog/2018-10-04-debezium-0-9-0-alpha2-released.adoc]: You mean grammatically or contents-wise? Can you give the alternative? Thanks!

Comment (jpechane) [reply blog/2018-10-04-debezium-0-9-0-alpha2-released.adoc]: Did not you intend to write "In terms of version upgrades ***we*** migrated"?

Comment (gunnarmorling) [reply blog/2018-10-04-debezium-0-9-0-alpha2-released.adoc]: Ah yes, of course :)

**Pull request closed without merging**